cmd/release/config/config.go

```diff
@@ -41,6 +41,25 @@ type RancherRelease struct {
 	SkipStatusCheck      bool   `json:"skip_status_check"`
 }
 
+type UIRelease struct {
+	UIRepoOwner   string `json:"ui_repo_owner" validate:"required"`
+	UIRepoName    string `json:"ui_repo_name"`
+	PreviousTag   string `json:"previous_tag"`
+	ReleaseBranch string `json:"release_branch" validate:"required"`
+	DryRun        bool   `json:"dry_run"`
```

Comment:
Despite using `dry_run` as a config in other places, I think using the global flag defined in `cmd/release/cmd/root.go` is way more convenient and better and easier to check if it's on or not

Reply:
If we try to directly use the `DryRun` flag in other places it will create an import cycle.